port_ocean/core/handlers/webhook/processor_manager.py

```diff
@@ -72,17 +70,29 @@ async def process_queue(self, path: str) -> None:
         """Process events for a specific path in order"""
         while True:
             matching_processors: list[AbstractWebhookProcessor] = []
-            event: WebhookEvent | None = None
+            webhookEvent: WebhookEvent | None = None
             try:
-                event = await self._event_queues[path].get()
-                with logger.contextualize(webhook_path=path, trace_id=event.trace_id):
-                    matching_processors = self._extract_matching_processors(event, path)
-                    await asyncio.gather(
-                        *(
-                            self._process_single_event(processor, path)
-                            for processor in matching_processors
+                queue = self._event_queues[path]
+                webhookEvent, eventContext = await queue.get()
+                with logger.contextualize(
+                    webhook_path=path, trace_id=webhookEvent.trace_id
+                ):
+                    async with event_context(
+                        EventType.LIVE_EVENT,
+                        trigger_type="machine",
+                        parent_override=eventContext,
+                    ):
+                        matching_processors = self._extract_matching_processors(
+                            webhookEvent, path
+                        )
+                        webhookEventDatas = await asyncio.gather(
```

Comment:
`webhookEventDatas` doesn't sounds right :/ I would suggest `webhook_event_results`
Also note that you are using camelCase instead of snake_case which is the python convention for setting parameters

Reply:
how are you handling failures here, we want to make sure that if one of the processors has failed we are still processing the others, but probably skipping any deletion.

Reply:
webhook event is missing the case where the integration return to the core that the resource is no longer in existence and requires deletion by the core